core/tauri/src/scope/shell.rs

```diff
@@ -71,13 +90,35 @@ pub struct ScopeConfig {
 #[derive(Debug, Clone)]
 pub struct ScopeAllowedCommand {
   /// The shell command to be called.
-  pub command: std::path::PathBuf,
+  command: PathBuf,
 
   /// The arguments the command is allowed to be called with.
-  pub args: Option<Vec<ScopeAllowedArg>>,
+  args: Option<Vec<ScopeAllowedArg>>,
 
   /// If this command is a sidecar command.
-  pub sidecar: bool,
```

Comment:
this is a breaking change, any reason to hide these?

Reply:
This struct was only used on codegen, no public API returned or takes it as input. I've hidden this so no one use it instead of the builder, which is the stable API.

Reply:
It still shows in documentation https://docs.rs/tauri/latest/tauri/scope/struct.ShellScopeAllowedCommand.html so even if no functions take it as input, we can't break it